common/model/exhibitions.js

```diff
@@ -38,7 +38,6 @@ export type Exhibition = {|
 export type UiExhibition = {|
   ...Exhibition,
   ...{|
-    galleryLevel: number, // this should be deprecated for place
```

Comment:
This is one of the nice things about TypeScript: it spotted that this isn't on the PrismicExhibitionDocument model, or indeed in our Prismic model at all – so we can delete it everywhere.